griptape/tools/date_time/tool.py

```diff
@@ -34,6 +35,9 @@ def get_relative_datetime(self, params: dict) -> BaseArtifact:
             date_string = params["values"]["relative_date_string"]
             relative_datetime = parse(date_string)
 
-            return TextArtifact(str(relative_datetime))
+            if not relative_datetime:
```

Comment:
Sorry to nit more but let's also make sure `if-else` structures read "naturally" with a positive `if` coming first and a negative being naturally tucked under `else`. Makes it a little easier to follow the code when developers don't have to negate a condition in their head while reading the code.

Reply:
Makes sense, updated!